src/backend/distributed/citus--7.0-8--7.0-9.sql

```diff
@@ -0,0 +1,73 @@
+/* citus-7.0-8--7.0-9 */
+
+SET search_path = 'pg_catalog';
+
+ALTER TABLE pg_dist_node ADD COLUMN nodecluster varchar(64) NOT NULL DEFAULT 'default';
```

Comment:
Actually, probably better to use the `name` type here, which ensures that the characters will fit in an array of NAMEDATALEN.